lib/json_web_token.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class JsonWebToken
+  SECRET_KEY = Rails.application.secrets.secret_key_base.to_s
+  def self.encode(payload, exp = 24.hours.from_now)
+  payload[:exp] = exp.to_i
+  JWT.encode(payload, SECRET_KEY)
+  end
+
+  def self.decode(token)
```

Comment:
What do you think about using the `class << self   ...  end` syntax. As it is now, it seems like a whole method instead of one constant and two definitions.